DDR4 OC Guide.md

```diff
@@ -525,11 +525,11 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
 
 8. Set `tRC = tRP + tRAS`. Increase if unstable.
    * tRC is only available on AMD and some Intel UEFIs.
-   * On Intel UEFIs, tRC does seem to be affected by tRP and tRAS, even if it is hidden.
+   * tRC as a timing is only aviable on AMD. On Intel this timing is not used so the effective tRC is tRAS + tRP or tRCD + tRTP + tRP, depending on which is higher. Due to this there can be many cases where stability issues are encountered due to the effective tRC being too low, as shown in the following images.
```

Comment:
```suggestion
   * tRC as a timing is now deprecated on Intel. Although it is present in certain bioses, including a lot of X299 bioses, it is no longer used and is either unchangeable directly or is not used when changed. On newer Intel platforms this timing is not used so the effective tRC is `tRAS + tRP` or `tRCD + tRTP + tRP` (the one of greater value is used). Due to this there can be many cases where stability issues are encountered due to the effective tRC being too low, as shown in the following images.
```

Reply:
It would be more accurate to call tRC a deprecated option on Intel. I remember it being maybe x79/x99 or similar having tRC as a modifiable option which was removed with Skylake and Skylake-X. Unless it did nothing back then, in which case strange. I bring this up as tRC is commonly an option on x299 bios as a holdover iirc